### PR TITLE
PWGCF: FemtoUniverse -- Fixing DetaDphi* cut

### PR DESCRIPTION
**Pull request by zchochul**
Before: Common delta eta delta phi* plots for same and mixed events (sum).

After: Separate plots for same and mixed, allowing you to divide the plots later and get a useful QA plot!

### CODE REVIEW

File: PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h [hunk 1]
```diff
@@ -197,13 +201,87 @@ class FemtoUniverseDetaDphiStar
...
+        } else {
+          histdetadpimixed[i][1]->Fill(deta, dphiAvg);
+        }
+      }
+      return pass;
+    } else {
+      LOG(fatal) << "FemtoUniversePairCleaner: Combination of objects not defined - quitting!";
+      return false;
+    }
+  }
```
Comment (victor-gonzalez) [inline PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h:272]: Basically you are repeating the full code
I would suggest either template on the kind of pair origin, same or mixed, or pass an additional parameter with such origin.
Then you rely on that origin to peek the proper histograms

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
I believe it is worth to consider my suggestion

File: PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h (hunk 1 above)
Comment (zchochul) [reply PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h:272]: Thank you! I will do that ;)

**Review by victor-gonzalez — COMMENTED** [review]
I would not use a `string` for coding same/mixed
Have in mind that you usually are in the pair loop, two inner loops, and you need to compare each character of the string
I would suggest using an `enum`, `kSame`, `kMixed`, (you probably already have something like that in the source/include hierarchy) with which you just compare one single value that is by far much more efficient

**Pull request closed without merging**